pkgs/top-level/python-aliases.nix

```diff
@@ -482,6 +482,7 @@ mapAliases ({
   pytest-pep8 = pytestpep8; # added 2021-01-04
   pytest-pep257 = throw "pytest-pep257 was removed, as the pep257 package was migrated into pycodestyle"; # added 2022-04-12
   pytest-pythonpath = throw "pytest-pythonpath is obsolete as of pytest 7.0.0 and has been removed"; # added 2022-03-09
+  pytest-runner = throw "pytest-runner has been removed as it uses deprecated features of setuptools and is deprecated by upstream"; # added 2024-09-15
```

Comment:
Why was it necessary to prevent this from being used, instead of just warning and removing all existing usage?

Reply:
Because we always add aliases for removed packages that have no version suffix. Otherwise users get an harder to debug undefined variable error.